core-services/prow/02_config/openshift/cluster-api-provider-alibaba/_pluginconfig.yaml

```
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cluster-api-provider-alibaba
+  require_self_approval: false
```

We tend to require self approval on our team's repos
```suggestion
  require_self_approval: true
```